smoketest/scripts/cli/test_vpn_ipsec.py

```diff
@@ -106,6 +107,32 @@
 CERT_PATH   = f'{swanctl_dir}/x509/'
 CA_PATH     = f'{swanctl_dir}/x509ca/'
 
+
+def _encode_to_base64(input_string):
```

Comment:
Probably a good candidate for `vyos.utils.convert`?

Reply:
It probably is, we can move it there is more use cases for it arise for sure.